UPTEthereumSigner/UPTHDSigner.m

```diff
@@ -71,7 +76,7 @@ + (void)showSeed:(NSString *)rootAddress prompt:(NSString *)prompt callback:(UPT
     {
         NSError *protectionLevelError = [[NSError alloc] initWithDomain:@"UPTHDError"
                                                                    code:UPTHDSignerErrorCodeLevelPrivateKeyNotFound.integerValue
-                                                               userInfo:@{@"message": @"private key not found for eth address"}];
+                                                               userInfo:@{@"message": [NSString stringWithFormat:@"private key not found for eth address: showSeed %@", rootAddress]}];
```

Comment:
Same as above. Also fix missing spaces; applies for other similar places above & below.